python/graphscope/analytical/app/pagerank.py

```diff
@@ -42,14 +42,50 @@ def pagerank(graph, delta=0.85, max_round=10):
 
     .. code:: python
 
-        import graphscope as gs
-        g = gs.g()
-        # Load some data, then project to a simple graph (if needed).
-        pg = g.project(vertices={"vlabel": []}, edges={"elabel": []})
-        r = gs.pagerank(pg, delta=0.85, max_round=10)
-        s.close()
-
+        >>> import graphscope
+        >>> from graphscope.dataset import load_p2p_network
+        >>> sess = graphscope.session(cluster_type="hosts", mode="eager")
+        >>> g = load_p2p_network(sess)
+        >>> # project to a simple graph (if needed)
+        >>> pg = g.project(vertices={"host": ["id"]}, edges={"connect": ["dist"]})
+        >>> c = graphscope.pagerank(pg, delta=0.85, max_round=10)
+        >>> sess.close()
     """
     delta = float(delta)
     max_round = int(max_round)
     return AppAssets(algo="pagerank", context="vertex_data")(graph, delta, max_round)
+
+
+@project_to_simple
+@not_compatible_for("arrow_property", "dynamic_property")
+def pagerank_nx(graph, alpha=0.85, max_iter=100, tol=1e-06):
+    """Evalute PageRank on a graph in NetworkX version.
```

Comment:
Evaluate pagerank on a graph using algorithm that exactly follows the implemented in the NetworkX library.